### PR TITLE
refactor(cache): Remove namespace param from decorator

### PR DESCRIPTION
**Pull request by zharinov**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/datasource/docker/index.ts [hunk 1]
```diff
@@ -221,12 +220,11 @@ export class DockerDatasource extends Datasource {
   }
 
   @cache({
-    namespace: 'datasource-docker-imageconfig',
     key: (
       registryHost: string,
       dockerRepository: string,
       configDigest: string,
-    ) => `${registryHost}:${dockerRepository}@${configDigest}`,
+    ) => `imageconfig|>${registryHost}:${dockerRepository}@${configDigest}`,
```
Comment (viceice) [inline lib/modules/datasource/docker/index.ts:227]: it's probably a bad idea to use same cache key again 🤔

**Comment by viceice** [comment]
added breaking labels, because cache time override is now only possible on datasource and no longer for functions.was heavily use by the app to reduce docker hub rate limiting

File: lib/modules/datasource/docker/index.ts (hunk 1 above)
Comment (zharinov) [reply lib/modules/datasource/docker/index.ts:227]: Yep, it's better to use new keys, but then we'll nuke a lot of data... Maybe it's better to refactor cache keys first

Comment (zharinov) [reply lib/modules/datasource/docker/index.ts:227]: Actually, we could leave long-term cache intact, so it's relatively safe to reset

**Pull request closed without merging**